tests/test_requirements.py

```diff
@@ -36,6 +36,7 @@ def teardown_method(self, method):
     @patch("homeassistant.util.package.is_virtual_env", return_value=True)
     @patch("homeassistant.util.package.is_docker_env", return_value=False)
     @patch("homeassistant.util.package.install_package", return_value=True)
+    @patch.dict(os.environ, {}, clear=True)
```

Comment:
I do think it is better, however, this clears the whole environment. What we actually want is to just clear the `WHEELS_LINKS` environment var.

At this point, it doesn't really matter, but it might in the future.

Reply:
Fixed

Reply:
Better 👍